packages/zen-bull-netting/test/unit-test/SetOTCPriceTolerance.t.sol

```diff
@@ -46,9 +46,8 @@ contract SetOTCPriceTolerance is ZenBullNettingBaseSetup {
     }
 
     function testSetOTCPriceToleranceWhenPeriodLessThanMinimum() public {
-        uint256 maxOtcPriceTolerance = zenBullNetting.MAX_OTC_PRICE_TOLERANCE();
         vm.prank(owner);
         vm.expectRevert(bytes("ZBN02"));
-        zenBullNetting.setOTCPriceTolerance(maxOtcPriceTolerance + 1);
+        zenBullNetting.setOTCPriceTolerance(3e17);
```

Comment:
why not max+1?

Reply:
I made it private, to decrease contract size, so can't fetch it, trying to do with Vm though